roles/st2mistral/tasks/main.yml

```diff
@@ -55,9 +42,29 @@
     - restart mistral
   tags: st2mistral
 
-- name: Setup Mistral DB tables, etc
+- name: Make sure "Initiate mistral database" doesn't run twice
```

Comment:
See: `creates: /etc/mistral/init_mistral_db.SQL.ansible.has.run` in previous task.
If file from the `creates` exists, - the command won't be executed.

See: http://docs.ansible.com/ansible/shell_module.html
for more info